src/utils/number.ts

```diff
@@ -26,20 +26,25 @@ export function parseNumberString(numberString?: string): string {
 }
 
 export function sanitizeDecimalString(decimalString: string): string {
-  return decimalString?.endsWith(".") ? decimalString.replace(".", "") : decimalString;
+  return decimalString.replace(/\.$/, "");
```

Comment:
Do you need to handle `decimalString` being `null`? Not sure if the previous version did as an extra precaution or an actual scenario. cc @eriklharper

Reply:
No because of #3627

Reply:
Noice. 👍🏼